pkg/rkenodeconfigserver/nodeserver.go

```diff
@@ -201,7 +202,7 @@ func (n *RKENodeConfigServer) nodeConfig(ctx context.Context, cluster *v3.Cluste
 				nc.Processes = enhanceWindowsProcesses(tempNode.Processes)
```

Comment:
Is the kubelet param needed by windows workers?

Reply:
Updated.